PennMobile/New Group/AccountPageViewController.swift

```diff
@@ -0,0 +1,132 @@
+//
+//  AccountPageViewController.swift
+//  PennMobile
+//
+//  Created by Andrew Antenberg on 9/26/21.
+//  Copyright © 2021 PennLabs. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class AccountPageViewController: UIViewController, ShowsAlertForError, UITableViewDelegate, UITableViewDataSource {
+    var account: Account!
+    let tableView = UITableView(frame: .zero, style: .insetGrouped)
+    
+    var profileInfo = [(text: "Name", info: " "), (text: "Username", info: " "), (text: "Email", info: " ")]
+    var educationInfo = [(text: "Graduation Term", info: " "), (text: "School", info: " "), (text: "Major", info: " ")]
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        setupView()
+        setupTableView()
+        guard Account.isLoggedIn else {
+            self.showAlert(withMsg: "Please login to use this feature", title: "Login Error", completion: { self.navigationController?.popViewController(animated: true)} )
+            return
+        }
+        setupProfileInfo()
+        setupEducationInfo()
+    }
+    
+    func setupView() {
+        self.title = "Account"
+        view.backgroundColor = .uiGroupedBackground
+        account = Account.getAccount()
+    }
+    func setupTableView() {
+        view.addSubview(tableView)
+        tableView.register(ProfilePageTableViewCell.self, forCellReuseIdentifier: ProfilePageTableViewCell.identifier)
+
+        tableView.translatesAutoresizingMaskIntoConstraints = false
+        tableView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor).isActive = true
+        tableView.leadingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leadingAnchor).isActive = true
+        tableView.trailingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.trailingAnchor).isActive = true
+        tableView.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor).isActive = true
+        tableView.rowHeight = UITableView.automaticDimension
+        tableView.estimatedRowHeight = 600
+        tableView.delegate = self
+        tableView.dataSource = self
+    }
+    
+    func setupProfileInfo() {
+        guard let firstName = account.first, let lastName = account.last else {
+            return
+        }
+        profileInfo[0].info = "\(firstName) \(lastName)"
```

Comment:
It would be easier for the reader if you instead did
`profileInfo.append((text: "name", info: "\(firstName) \(lastName)"))`
so the value of the text and info are in the same place. If we need to quickly add a new entry, this way we don't need to edit two separate locations in the file.